cli/src/main.rs

```diff
@@ -6,7 +6,7 @@ use base64;
 use clap::{crate_name, crate_version, App, AppSettings, SubCommand};
 use serde_json::json;
 
-use anycloud::deploy::{client_error, get_config, info, new, terminate, upgrade, ALAN_VERSION};
+use anycloud::deploy;
```

Comment:
Please check if with this changes everything compiles and `error!` macro still working because it need to have `client_error` imported and I'm not sure if it is able to find it if it is not explicitly imported, haven't tried that

Reply:
if this does break everything, the `error!` macro's `client_error` call can instead be `::anycloud::deploy::client_error` and we can get rid of `client_error` imports everywhere

(might be `crate::deploy::client_error`, `::deploy::client_error`, or something like that - I forget off the top of my head)

Reply:
> if this does break everything, the `error!` macro's `client_error` call can instead be `::anycloud::deploy::client_error` and we can get rid of `client_error` imports everywhere
> 
> (might be `crate::deploy::client_error`, `::deploy::client_error`, or something like that - I forget off the top of my head)

nice